go.mod

```diff
@@ -4,55 +4,54 @@ go 1.11
 
 require (
 	github.com/btcsuite/winsvc v1.0.0
-	github.com/decred/base58 v1.0.0
-	github.com/decred/dcrd/addrmgr v1.0.2
-	github.com/decred/dcrd/blockchain/stake/v2 v2.0.1
-	github.com/decred/dcrd/blockchain/standalone v1.0.0
-	github.com/decred/dcrd/blockchain/v2 v2.0.2
+	github.com/decred/base58 v1.0.1
+	github.com/decred/dcrd/addrmgr v1.1.0
+	github.com/decred/dcrd/blockchain/stake/v2 v2.0.2
+	github.com/decred/dcrd/blockchain/standalone v1.1.0
+	github.com/decred/dcrd/blockchain/v2 v2.1.0
 	github.com/decred/dcrd/certgen v1.1.0
 	github.com/decred/dcrd/chaincfg/chainhash v1.0.2
 	github.com/decred/dcrd/chaincfg/v2 v2.3.0
-	github.com/decred/dcrd/connmgr/v2 v2.0.0
+	github.com/decred/dcrd/connmgr/v2 v2.1.0
 	github.com/decred/dcrd/crypto/ripemd160 v1.0.0
-	github.com/decred/dcrd/database/v2 v2.0.0
+	github.com/decred/dcrd/database/v2 v2.0.1
 	github.com/decred/dcrd/dcrec v1.0.0
 	github.com/decred/dcrd/dcrec/secp256k1/v2 v2.0.0
-	github.com/decred/dcrd/dcrjson/v3 v3.0.0
-	github.com/decred/dcrd/dcrutil/v2 v2.0.0
+	github.com/decred/dcrd/dcrjson/v3 v3.0.1
+	github.com/decred/dcrd/dcrutil/v2 v2.0.1
 	github.com/decred/dcrd/fees/v2 v2.0.0
-	github.com/decred/dcrd/gcs/v2 v2.0.0
-	github.com/decred/dcrd/hdkeychain/v2 v2.0.1
+	github.com/decred/dcrd/gcs/v2 v2.0.1
+	github.com/decred/dcrd/hdkeychain/v2 v2.1.0
 	github.com/decred/dcrd/lru v1.0.0
 	github.com/decred/dcrd/mempool/v3 v3.1.0
-	github.com/decred/dcrd/mining/v2 v2.0.0
-	github.com/decred/dcrd/peer/v2 v2.0.0
+	github.com/decred/dcrd/mining/v2 v2.0.1
+	github.com/decred/dcrd/peer/v2 v2.1.0
 	github.com/decred/dcrd/rpc/jsonrpc/types/v2 v2.0.0
 	github.com/decred/dcrd/rpcclient/v5 v5.0.0
-	github.com/decred/dcrd/txscript/v2 v2.0.0
+	github.com/decred/dcrd/txscript/v2 v2.1.0
 	github.com/decred/dcrd/wire v1.3.0
-	github.com/decred/dcrwallet/rpc/jsonrpc/types v1.2.0
-	github.com/decred/go-socks v1.0.0
+	github.com/decred/dcrwallet/rpc/jsonrpc/types v1.3.0
+	github.com/decred/go-socks v1.1.0
 	github.com/decred/slog v1.0.0
-	github.com/gorilla/websocket v1.4.0
+	github.com/gorilla/websocket v1.4.1
```

Comment:
This seems to be the only non-decred repo updated directly.  The changes aren't breaking of course, but they aren't trivial either: https://github.com/gorilla/websocket/compare/v1.4.0...v1.4.1

But dcrdata is still on v1.4.0 so I can't testify for v1.4.1 other than to say that our very limited use of v1.4.1 in dcrdex has been OK so far.

Reply:
dcrwallet is also using v1.4.1.

Reply:
Ah, sorry I should have commented in https://github.com/decred/dcrd/pull/1946

Reply:
Anyway, it's the beginning of the release cycle so this is the time.  I just wanted to point out the change since wasn't mentioned here or in https://github.com/decred/dcrd/pull/1946

Reply:
Good point.  I should've mentioned that over in #1946.  However, I did review the changes before I bumped the version.

Reply:
I updated its PR description to call that out.